packages/table/preview/index.tsx

```diff
@@ -644,6 +646,13 @@ class MutableTable extends React.Component<{}, IMutableTableState> {
         this.maybeLogCallback(`[onCopy] success = ${success}`);
     }
 
+    private onVisibleCellsChange = (rowIndices: IRowIndices, columnIndices: IColumnIndices) => {
+        const { rowIndexStart, rowIndexEnd } = rowIndices;
+        const { columnIndexStart, columnIndexEnd } = columnIndices;
+        this.maybeLogCallback(`[onVisibleCellsChange] rowIndexStart=${rowIndexStart} rowIndexEnd=${rowIndexEnd}` +
```

Comment:
Yeah, it includes both the newline and the indentation:
![image](https://user-images.githubusercontent.com/443450/27602362-5f240c82-5b27-11e7-8aa4-7daec66b59e6.png)